src/styles/blocks/page.css

```diff
@@ -0,0 +1,14 @@
+.page {
```

Comment:
it's better to set for the whole `html`
```suggestion
html {
```